docs/README.md

```diff
@@ -0,0 +1,132 @@
+# Introduction
+
+> These pages are automatically generated from content hosted in this [GitHub repository](https://github.com/DEFRA/software-development-standards)
+
+## About
+
+This is the home for standards and guidance relating to software development in [Defra](https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs).
+
+These form part of the software development standards policy for Defra and apply to all software development undertaken on behalf of the Defra group whether by staff, contingent workers or suppliers.
+
+All developers working on behalf of Defra need to be aware of their obligation to follow these standards.
+
+Any deviation must be managed as an exception under Defra's architectural governance processes.
+
+We maintain these standards on GitHub to ensure they have the greatest visibility to all developers and are readily available in a familiar environment. To aid readability, you can refer to the [GitHub documentation on managing accessibility settings](https:/..github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/managing-accessibility-settings).
+
+## Principles
+
+The standards and guidance provided here a based on an agreed set of principles.
+
+- [Principles for these standards](./principles/README.md)
+- [Coding principles](./principles/coding_principles.md)
+- [Security principles](./principles/security_principles.md)
+
+
+## Standards
+
+These are the standards that must be followed for all software development undertaken on behalf of the Defra group.
+
+- [List of all standards (A-Z)](./standards/README.md)
+
+### Coding standards
+
+- [Common coding standards](./standards/common_coding_standards.md)
+- [Development languages](./standards/development_language_standards.md)
+- [JavaScript standards](./standards/javascript_standards.md)
+- [C# coding standards](./standards/csharp_coding_standards.md)
+- [Mobile application standards](./standards/mobile_app_standards.md)
+- [Node.js standards](./standards/node_standards.md)
+- [UiPath standards](./standards/uipath_standards.md)
+- [Quality assurance and test standards](./standards/quality_assurance_standards.md)
+- [Security standards](./standards/security_standards.md)
+- [Version control standards](./standards/version_control_standards.md)
+
+### Packaging and deployment standards
+  
+- [Container standards](./standards/container_standards.md)
+- [Deployment standards](./standards/deployment_standards.md)
+- [Kubernetes standards](./standards/kubernetes_standards.md)
+
+### Database standards
+
+- [PL/SQL coding standards](./standards/plsql_coding_standards.md)
+- [TSQL and SQL Server database standards](./standards/tsql_and_sqldb_standards.md)
+- [README standards](./standards/readme_standards.md)
```

Comment:
Should we move README standards out of the database standards section?

Reply:
Yeah definitely 😆 that looks like it got demoted when the conversion was being done - it's a top-level item on the existing page, but on this new page they've all been converted into headings.

It probably needs its own section, maybe "Documentation standards"?

Reply:
Ha! my bad.

Moved under it's own `Documentation standards" header as suggested.